### PR TITLE
Fix: Bump `eigensdk-go` version

### LINKED ISSUE #1468: fix: Bump eigensdk-go version
We currently use version `v1.1.12` we should bump to `v1.1.13`

### PR DESCRIPTION
**Pull request by PatStiles**
# Bump eigensdk-go version 

## Description

Closes #1468 . Bumps the `eigensdk-go` version from `v1.1.12` -> `v1.1.13` . 

Note, `v1.1.12`  contained a library for parsing .yaml and .json files. This library was used in Aligned for starting the variouscomponents of the network. This library was removed in `v1.1.13`. To work around this breaking change the library was re-implemented within the Aligned Layer repo within `core/utils/config_utils.go`, the library was ported from this [branch](https://github.com/Layr-Labs/eigensdk-go/blob/v0.1.12_blsagg_window/utils/utils.go`) of the eigen-sdk repository.

## To Test:
- Run a local devnet and verify proofs land:
1. `make anvil_start_with_block_time`
2. `make aggregator_start`
3. `make operator_full_registration CONFIG_FILE=config-files/config-operator-1.yaml` 
4. `make operator_start CONFIG_FILE=config-files/config-operator-1.yaml`
5. `make batcher_start_local`
6. `make telemetry_full_start`

Once you have the system running, send proofs

7. `make batcher_send_burst_groth16`
8. Go to jaeger `http://localhost:16686/search`
9. You should see the traces correctly created and task verified
image

## Type of change

Please delete options that are not relevant.

- [x] New feature

## Checklist

- [x] Linked to Github Issue

### CODE REVIEW

File: core/utils/config_utils.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	TextCharsLimit = 500
+)
+
+var (
+	// ImageExtensions List of common image file extensions
+	// Only support PNG for now to reduce surface area of image validation
+	// We do NOT want to support formats like SVG since they can be used for javascript injection
+	// If we get pushback on only supporting png, we can support jpg, jpeg, gif, etc. later
+	ImageExtensions = []string{".png"}
+)
```
Comment (MarcosNicolau) [inline core/utils/config_utils.go]: Do we need this?

**Review by MarcosNicolau — APPROVED** [review]
Everything works fine, tested on macos.

File: core/utils/config_utils.go (hunk 1 above)
Comment (PatStiles) [reply core/utils/config_utils.go]: v1.1.12 included utility functions for parsing JSON and YAML files which we used. There were eliminated in v1.1.13, I reimplemented them within our library.

Comment (MarcosNicolau) [reply core/utils/config_utils.go]: Yes but I mean the constants and ImageExtensions.

Comment (MauroToscano) [reply core/utils/config_utils.go]: I don't think we need the image part of the code

File: core/utils/config_utils.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+func ReadFile(path string) ([]byte, error) {
+	b, err := os.ReadFile(filepath.Clean(path))
+	if err != nil {
+		return nil, err
+	}
+	return b, nil
```
Comment (Oppen) [inline core/utils/config_utils.go]:
```suggestion
	return os.ReadFile(filepath.Clean(path))
```

File: core/utils/config_utils.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+}
+
+func ReadYamlConfig(path string, o interface{}) error {
+	if _, err := os.Stat(path); errors.Is(err, os.ErrNotExist) {
+		log.Fatal("Path ", path, " does not exist")
+	}
+	b, err := ReadFile(path)
+	if err != nil {
+		return err
+	}
```
Comment (Oppen) [inline core/utils/config_utils.go]: Just let the `ReadFile` fail:
```suggestion
	b, err := ReadFile(path)
	if err != nil {
		return err
	}
```

Comment (Oppen) [reply core/utils/config_utils.go]: If you would rather decorate and log the error:
```suggestion
	if err != nil {
		log.Fatalf("%s: %e", path, err)
	}
```
Note if an error reading breaks our config we should crash just the same.

File: core/utils/config_utils.go (hunk 1 above)
Comment (Oppen) [reply core/utils/config_utils.go]: I agree, the consts and the var should go.

Comment (MauroToscano) [reply core/utils/config_utils.go]: Solved

**Review by MauroToscano — APPROVED** [review]

**Review by MarcosNicolau — APPROVED** [review]
Re-tested with new changes.

**Review by uri-99 — APPROVED** [review]

**Pull request merged**